input/input.go

```diff
@@ -41,6 +42,18 @@ type Input interface {
 	// HeightHint returns the minimum height at which a confirmed spending
 	// tx can occur.
 	HeightHint() uint32
+
+	// Cpfp returns information about a possibly unconfirmed parent tx.
+	Cpfp() *TxInfo
```

Comment:
call this `ParentFeeInfo` or something? I found having it named `CPFP` a bit confusing.

Reply:
I had that first, but the reason I renamed it is that it isn't just the parent info, but also signals specific cpfp behavior. Like when the input is included in a sweep.

Reply:
Renamed to `UnconfParent()`. The comments already says that that is what it is.